atlas/portal/api/portal.api.proto

```diff
@@ -0,0 +1,126 @@
+syntax = "proto3";
+package portal.pb;
+option go_package = "atlas/portal/pkg/pb";
+
+import "google/api/annotations.proto";
+import "google/protobuf/empty.proto";
+
+service Portal{
+  rpc GetVersion (google.protobuf.Empty) returns (VersionResponse) {
```

Comment:
If you have no need of this endpoint, please remove it.

Reply:
Agree. All unused/useless/commented code should be removed

Reply:
Done